### PR TITLE
US: Updates

### LINKED ISSUE #3545: Fix: Gunaz TV
**_Channel Name:_** Gunaz TV
**_Broken Link (from playlist):_** http://gtv.live.cdn.bitgravity.com/gtv/live/feed03
**_Possible Replacement (optional):_** https://gtv.live-s.cdn.bitgravity.com/cdn-live/_definst_/gtv/live/multibitrate.smil/chunklist_w1484311239_b450000.m3u8
**_Notes (optional):_** Attempts to connect, but times out in VLC, FFplay, Kodi.

There is another channel in the us.m3u playlist with similar name.  (Günaz)  It does finally streams with no audio.

Website: http://gunaz.live/

Check other sites, it will not play there also. 

Gives 404 error on this site. (https://streema.com/tv/GunazTV)

Günaz Tv Canlı Yayını does play at this address: https://canlitv.com/gunaz-tv 

This site has a lot of channels: https://canlitv.com/tr

### LINKED ISSUE #3715: Add: Nick East
**Channel Name:** Nick East
**Country:** USA
**Language:** English
**Website:** 
**Stream URL (optional):** http://31.220.41.88:8081/live/us-nick.stream/playlist.m3u8
**Notes (optional):** Works in GSE Smart IPTV and Pot Player

### LINKED ISSUE #3733: Add: Pennsylvania Cable Network
PCN Pennsylvania Cable Net

**Channel Name:** PCN Pennsylvania Cable Net 
**Country:** US
**Language:** English
**Website:** bozztv.com
**Stream URL (optional):** https://bozztv.com/teleyupp/teleup-pcnhd/tracks-v2a1/mono.m3u8
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by Dum4G**
Reinstating https://github.com/iptv-org/iptv/pull/3734 for clean merge process
Closes #3733
Closes https://github.com/iptv-org/iptv/issues/3715
Closes https://github.com/iptv-org/iptv/issues/3545

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1740,6 +1738,8 @@ https://gustous-plex.amagi.tv/hls/amagi_hls_data_gustoAAAA-gustous-plex/CDN/play
 https://dai.google.com/linear/hls/event/tdSCy5u2R5WtCLXX4NwDtg/master.m3u8
 #EXTINF:-1 tvg-id="GunAzTV.us" tvg-name="GünAz TV" tvg-country="US;AZ" tvg-language="Azerbaijani" tvg-logo="" group-title="General",GünAz TV
 rtsp://gtv.live-s.cdn.bitgravity.com/cdn-live/_definst_/gtv/live/feed02?nc=1
+#EXTINF:-1 tvg-id="GunAzTV.us" tvg-name="GünAz TV" tvg-country="US;AZ" tvg-language="Azerbaijani" tvg-logo="" group-title="General",GünAz TV
+http://gtv.live.cdn.bitgravity.com/gtv/live/feed03
```
Comment (LaneSh4d0w) [inline channels/us.m3u:1742]: Hmm... why do I get a 1gb flv file on this one ?

Comment (Dum4G) [reply channels/us.m3u:1742]: Looks like a legit broadcast to me

Comment (LaneSh4d0w) [reply channels/us.m3u:1742]: It just sounds weird to get a fixed file. I'm gonna take a look.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by nickpythonluajava** [comment]
YOOOOO FINALLY

**Pull request merged**